backends/tfhe-cuda-backend/cuda/include/pbs/pbs_multibit_utilities.h

```diff
@@ -225,10 +225,12 @@ template <typename Torus> struct pbs_buffer<Torus, PBS_TYPE::MULTI_BIT> {
           num_blocks_keybundle * (polynomial_size / 2) * sizeof(double2),
           stream, gpu_index);
       global_accumulator = (Torus *)cuda_malloc_async(
```

Comment:
I think this global_accumulator needs to be kept as is for the "default" multibit PBS variant. Only for TBC and CG it should not be sized with levels, am I wrong?

Reply:
Ah yes, I just renamed it. It is used as the "join buffer" in `mul_ggsw_glwe`, so I'm calling it by the right name.